src/main/java/core/basesyntax/service/chosefruitoperationservice/FruitOperationHandler.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service.chosefruitoperationservice;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface FruitOperationHandler {
```

Comment:
I can't see real sense in this layer.
Your activities are enough to handle trasactions, additional strategy to get operation for another starategy is an overkill

Reply:
++